packages/cadl-rlc-test/test/authoring/cadl-output/src/models.ts

```diff
@@ -1,21 +1,29 @@
-/** The template for adding optional properties. */
-export interface OptionalPropertiesUpdateableProperties {
+export interface Project {
+  projectName: string;
   /**
    * The project kind.
    *
    * Possible values: CustomSingleLabelClassification, CustomMultiLabelClassification, CustomEntityRecognition
    */
-  projectKind?: string;
+  projectKind: string;
   /** The storage container name. */
-  storageInputContainerName?: string;
+  storageInputContainerName: string;
   /** The project settings. */
   settings?: Record<string, string>;
   /** Whether the project would be used for multiple languages or not. */
   multilingual?: boolean;
   /** The project description. */
   description?: string;
   /** The project language. This is BCP-47 representation of a language. For example, use "en" for English, "en-gb" for English (UK), "es" for Spanish etc. */
-  language?: string;
+  language: string;
+  /** Represents the project creation datetime. */
+  createdDateTime: Date | string;
+  /** Represents the project last modification datetime. */
+  lastModifiedDateTime: Date | string;
+  /** Represents the project last training datetime. */
+  lastTrainedDateTime: Date | string;
+  /** Represents the project last deployment datetime. */
+  lastDeployedDateTime: Date | string;
```

Comment:
interface ProjectMergeAndPatch = Partial<Exclude<Project, "">>;
interface ProjectPatch = Exclude<Project, "">;

Reply:
@joheredi I will file another PR to fix this issue.